src/styles/style.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
Following to the task descriptions you don't need to write the styles here, instead, you should use just .scss files so this is redundant

Reply:
Bump, think you don't need this file